TestPlaybooks/playbook-ReversingLabsTitaniumCloud-Test.yml

```diff
@@ -0,0 +1,216 @@
+id: 01f92c20-349f-4ced-877e-30476d014fe3
+version: 2
+name: ReversingLabsTitaniumCloud-Test
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: fed3f37d-24f9-4764-82dd-f53ed02a2b24
+    type: start
+    task:
+      id: fed3f37d-24f9-4764-82dd-f53ed02a2b24
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 50
+        }
+      }
+  "1":
+    id: "1"
+    taskid: 005acac6-3573-458f-8e89-414c661617cf
+    type: regular
+    task:
+      id: 005acac6-3573-458f-8e89-414c661617cf
+      version: -1
+      name: Extract File Hashes
+      scriptName: ExtractHash
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+      - "3"
+      - "4"
+    scriptarguments:
+      text: ${incident}
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 195
+        }
+      }
+  "2":
+    id: "2"
+    taskid: 15bba6b2-1259-4136-8e90-3cacc3bd2554
+    type: regular
+    task:
+      id: 15bba6b2-1259-4136-8e90-3cacc3bd2554
+      version: -1
+      name: Enrich MD5
+      script: ReversingLabs Titanium Core|||file
+      type: regular
+      iscommand: true
+      brand: ReversingLabs Titanium Core
```

Comment:
You mean `ReversingLabs Titanium Cloud`? and not `Core`

Reply:
Yes